MC/bin/o2dpg_sim_workflow_anchored.py

```diff
@@ -232,19 +232,22 @@ def retrieve_MinBias_CTPScaler_Rate(ctpscaler, finaltime, trig_eff, NBunches, Co
     if ColSystem == "PbPb":
       ctpclass = 25  # <--- we take scalers for ZDC
       ctptype = 7
-    print("Fetching rate with class " + str(ctpclass) + " type " + str(ctptype))
+    print("Fetching rate with time " + str(finaltime) + " class " + str(ctpclass) + " type " + str(ctptype))
     rate = ctpscaler.getRateGivenT(finaltime, ctpclass, ctptype)
     #if ColSystem == "PbPb":
     #  rate.first = rate.first / 28.
     #  rate.second = rate.second / 28.
 
     print("Global rate " + str(rate.first) + " local rate " + str(rate.second))
+    scaler = None
+    if rate.second >= 0:
+      scaler = rate.second
     if rate.first >= 0:
```

Comment:
@chiarazampolli do you know why we would only return the final/corrected local rate if the global rate is `rate.first >= 0`? I don't see it being used inside this branch.

Reply:
Was it maybe added here: https://github.com/AliceO2Group/O2DPG/pull/1333?